pysoa/common/transport/redis_gateway/core.py

```diff
@@ -194,7 +199,7 @@ def send_message(self, queue_name, request_id, meta, body, message_expiry_in_sec
         with self._get_timer('send.serialize'):
             serialized_message = self.serializer.dict_to_blob(message)
 
-        if len(serialized_message) > self.MAXIMUM_MESSAGE_BYTES:
+        if len(serialized_message) > self.maximum_message_size_in_bytes:
```

Comment:
could we additionally do
```
elif len(serialized_message) > (self.maximum_message_size_in_bytes * .9):
    log.warn('large message') # or something like that
```

(this could be future improvement, by no means blocking this change)